src/app/patterns/alerts/alerts.component.html

```diff
@@ -0,0 +1,63 @@
+<td-layout-nav logo="assets:teradata" navigationRoute="/">
```

Comment:
Should the top bar say Dialogs instead of Alerts? 
![localhost_4200_ 2](https://user-images.githubusercontent.com/7193975/49904487-2c91e400-fe1f-11e8-9de2-efb51bc774cc.png)

Reply:
Yes and the button looks odd now. I'll take a look.